packages/block-editor/src/components/link-control/index.js

```diff
@@ -313,27 +416,49 @@ function LinkControl( {
 			ref={ wrapperNode }
 			className="block-editor-link-control"
 		>
-			{ isEditingLink || ! value ? (
+			{ isResolvingLink && (
+				<div className="block-editor-link-control__loading">
+					{ __( 'Loading' ) }...
```

Comment:
I'm not sure if there are any preference whether the ellipsis be included inside the translated string, but in any case: The ellipsis should be proper ellipsis character, not three dots.

Related:

https://github.com/WordPress/gutenberg/blob/77bd2748461bafcbea7d082aa6824e2473d7890c/.eslintrc.js#L88-L92

Here's the character for easy copying: `…`